transformations/correct_common_misspellings/transformation.py

```diff
@@ -0,0 +1,4117 @@
+        "priciple": "principle",
+        "receeding": "receding",
+        "broady": "broadly",
+        "oppenly": "openly",
+        "hismelf": "himself",
+        "hvae": "have",
+        "unsucesful": "unsuccessful",
+        "facilites": "facilities",
+        "mchanics": "mechanics",
+        "regularily": "regularly",
+        "hydropilic": "hydrophilic",
+        "circut": "circuit",
+        "Saterday": "Saturday",
+        "usefull": "useful",
+        "rhytmic": "rhythmic",
+        "mantained": "maintained",
+        "menally": "mentally",
+        "montains": "mountains",
+        "charachters": "characters",
+        "combinatins": "combinations",
+        "agains": "against",
+        "recomend": "recommend",
+        "femminist": "feminist",
+        "annuled": "annulled",
+        "soluable": "soluble",
+        "annouced": "announced",
+        "reccuring": "recurring",
+        "rendevous": "rendezvous",
+        "mroe": "more",
+        "substracts": "subtracts",
+        "coputer": "computer",
+        "poportional": "proportional",
+        "alledgedly": "allegedly",
+        "exeedingly": "exceedingly",
+        "newletters": "newsletters",
+        "sucesfuly": "successfully",
+        "compulsery": "compulsory",
+        "Champange": "Champagne",
+        "spreaded": "spread",
+        "intepretation": "interpretation",
+        "matheticians": "mathematicians",
+        "persistant": "persistent",
+        "harras": "harass",
+        "rewriet": "rewrite",
+        "enviroments": "environments",
+        "stateman": "statesman",
+        "shrinked": "shrunk",
+        "nuculear": "nuclear",
+        "sandwhich": "sandwich",
+        "copywrite": "copyright",
+        "miscelaneous": "miscellaneous",
+        "propperly": "properly",
+        "cxan": "cyan",
+        "Buddist": "Buddhist",
+        "bankrupcy": "bankruptcy",
+        "reasearch": "research",
+        "attened": "attended",
+        "unsuccesfully": "unsuccessfully",
+        "exinct": "extinct",
+        "expropiated": "expropriated",
+        "omniverously": "omnivorously",
+        "pamplet": "pamphlet",
+        "disatisfaction": "dissatisfaction",
+        "anomolies": "anomalies",
+        "sudents": "students",
+        "muder": "murder",
+        "scirpt": "script",
+        "occassionaly": "occasionally",
+        "incompatiblity": "incompatibility",
+        "usefuly": "usefully",
+        "consdiered": "considered",
+        "tradionally": "traditionally",
+        "magizine": "magazine",
+        "pinoneered": "pioneered",
+    }
+
+
+"""
+# code to process machine-readable text into this list (first save from wikipedia into dict.txt)
+import numpy as np
+
+# read everything in
+d = np.loadtxt("dict.txt", delimiter="->", dtype=str)
+
+# prune anything with commas
+unique_idxs = np.array([len(x.split()) == 1 for x in d[:, 1]])
+d = d[unique_idxs]
+
+# print as dict
+d = {d[i, 0]: d[i, 1] for i in range(d.shape[0])}
+print(d)
+"""
```

Comment:
I would remove this commented out code

Reply:
I'm keeping this since it shows some of the preprocessing that was done to clean the data